02_activities/assignments/assignment.sh

```diff
@@ -21,22 +21,27 @@ unzip rawdata.zip
 # Complete assignment here
 
 # 1. Create a directory named data
+mkdir data
 
 # 2. Move the ./rawdata directory to ./data/raw
-
+mv ./rawdata ./data/raw
 # 3. List the contents of the ./data/raw directory
-
+ls ./data/raw
 # 4. In ./data/processed, create the following directories: server_logs, user_logs, and event_logs
-
+cd data
+mkdir processed
+cd processed
+mkdir server_logs user_logs event_logs
 # 5. Copy all server log files (files with "server" in the name AND a .log extension) from ./data/raw to ./data/processed/server_logs
-
+cp data/raw/*server*.log data/processed/server_logs/
 # 6. Repeat the above step for user logs and event logs
-
+cp data/raw/*server*.log /data/processed/user_logs/
```

Comment:
what they meant was to grab the user logs and put them into the user_logs directory.